Signal/src/Models/TSMessageAdapaters/TSMessageAdapter.m

```diff
@@ -177,19 +177,9 @@ - (instancetype)initWithInteraction:(TSInteraction *)interaction
                     }
                 } else if ([attachment isKindOfClass:[TSAttachmentPointer class]]) {
                     TSAttachmentPointer *pointer = (TSAttachmentPointer *)attachment;
-                    adapter.messageType          = TSInfoMessageAdapter;
-
-                    switch (pointer.state) {
-                        case TSAttachmentPointerStateEnqueued:
-                            adapter.messageBody = NSLocalizedString(@"ATTACHMENT_QUEUED", nil);
```

Comment:
Did you happen to find out if we're queuing attachment downloads and if so what's the max number of simultaneous downloads?